src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -277,7 +277,7 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 
 			this._table = new Table<Slick.SlickData>(this._inputContainer.nativeElement, { dataProvider: this._tableData, columns: this._tableColumns }, options);
 			this._table.setData(this._tableData);
-			this._table.setSelectionModel(new RowSelectionModel({ selectActiveRow: true }));
+			this._table.setSelectionModel(new RowSelectionModel({ selectActiveRow: false }));
```

Comment:
what is this for?

Reply:
When select active row is true, everytime the a cell is focused the focus then goes to the entire row.
Example:
![azuredatastudio-insiders_uykTv5b1e9](https://user-images.githubusercontent.com/6816294/191129879-19816cae-9c6f-4933-9bee-86a04f17b889.gif)

After disabling it, the behavior becomes normal and expected.

Reply:
![azuredatastudio_v59tA7R7py](https://user-images.githubusercontent.com/6816294/191130205-a58de0e1-553d-4ecb-893a-f0e90c99b680.gif)